packages/vscode-extension/src/utils/commonUtils.ts

```diff
@@ -543,3 +537,19 @@ export function getTriggerFromProperty(args?: any[]) {
       return { [TelemetryProperty.TriggerFrom]: TelemetryTriggerFrom.Unknow };
   }
 }
+
+export function getTunnelingServiceFromVSCodeSettings(): TunnelingService {
+  const configuration: vscode.WorkspaceConfiguration =
+    vscode.workspace.getConfiguration(CONFIGURATION_PREFIX);
+  // The actual meaning of "prerequisiteCheck.ngrok" is whether tunneling is enabled.
```

Comment:
Will we finally update the display name in vsc settings? Showing `ngrok` to end users seems to be confused.

Reply:
Display name is already updated. 

<img width="377" alt="image" src="https://user-images.githubusercontent.com/9698542/170001223-60062d9f-358f-44b7-b835-50124992fb7d.png">

Reply:
But still have the `ngrok` wording, e.g., the key `xxx.ngrok` in .vscode/settings.json